packages/static-module-record/test/test-static-module-record.js

```diff
@@ -30,6 +32,14 @@ function assertDefaultExport(t, record) {
   t.deepEqual(record.__liveExportMap__, {});
 }
 
+function readFixture(filename) {
+  return fs.readFileSync(
+    new URL(filename, `${new URL(path.dirname(import.meta.url))}/fixtures/`)
+      .pathname,
```

Comment:
For the benefit of Windows in CI, something like:
```suggestion
    url.fileURLToPath(new URL(filename, new URL('fixtures/', import.meta.url)))
```

With
```js
import url from 'url';
```

Reply:
Added a commit to this effect.